CollectObject.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import awkward as ak
+
+class CollectObject:
+
+    def __init__(self, events):
+        self.events = events
+
+    def CreateMask(self, variable, cut):
+        loaded = self.objects[variable]
+        sign = cut[0]
+        value = cut[1]
+        if sign == "==":
+            self.mask = self.mask & (loaded == value)
+        elif sign == ">=":
+            self.mask = self.mask & (loaded >= value)
+        elif sign == ">":
+            self.mask = self.mask & (loaded > value)
+        elif sign == "<=":
+            self.mask = self.mask & (loaded <= value)
+        elif sign == "<":
+            self.mask = self.mask & (loaded < value)
+        elif sign == "!=":
+            self.mask = self.mask & (loaded != value)
+        else:
+            pass
```

Comment:
this part i am not so proud of but didn't find a way to better configure this in a generic way. any ideas?
but because dask already optimizes the pipeline, it won't disturb the computing performance too much